src/MovieCard.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './MovieCard.css';
+
+const MovieCard = ({ title, avgRating, photo, releaseDate }) => {
```

Comment:
I like that movie card is a functional component because it does not need to hold on to any data